org.eclipse.scanning.server/src/org/eclipse/scanning/server/servlet/ScanProcess.java

```diff
@@ -345,38 +345,38 @@ private IDeviceController createRunnableDevice(ScanBean bean, IPointGenerator<?>
 
 		ScanRequest<?> req = bean.getScanRequest();
 		if (req==null) throw new ScanningException("There must be a scan request to run a scan!");
-		
+
 		try {
 			final ScanModel scanModel = new ScanModel();
 			IPointGenerator<?> generator = getGenerator(req);
 			scanModel.setPositionIterable(generator);
-			
+
 			ScanEstimator estimator = new ScanEstimator(Services.getGeneratorService(), bean.getScanRequest());
 			bean.setSize(estimator.getSize());
 			scanModel.setFilePath(bean.getFilePath());
-			
+
 			scanModel.setDetectors(getDetectors(req.getDetectors()));
 			scanModel.setScannables(getScannables(getScannableNames(generator)));
 			scanModel.setMonitors(getScannables(req.getMonitorNames()));
 			scanModel.setScanMetadata(req.getScanMetadata());
 			scanModel.setBean(bean);
-			
+
 			ScanInformation scanInfo = new ScanInformation(estimator);
 			scanInfo.setFilePath(bean.getFilePath());
 			scanInfo.setScannableNames(getScannableNames(generator));
 			scanModel.setScanInformation(scanInfo);
-			
+
 			configureDetectors(req.getDetectors(), scanModel, generator);
-			
+
 			IPausableDevice<ScanModel> device = (IPausableDevice<ScanModel>) Services.getRunnableDeviceService().createRunnableDevice(scanModel, publisher, false);
 			IDeviceController controller = Services.getWatchdogService().create(device);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "controller" which hides the field declared at line 81. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)